lua/lvim/utils/hooks.lua

```diff
@@ -12,31 +12,22 @@ function M.run_pre_reload()
   Log:debug "Starting pre-reload hook"
 end
 
-function M.run_on_packer_complete()
-  Log:debug "Packer operation complete"
-  vim.api.nvim_exec_autocmds("User", { pattern = "PackerComplete" })
-
-  -- -- FIXME(kylo252): nvim-tree.lua/lua/nvim-tree/view.lua:442: Invalid window id
-  -- vim.g.colors_name = lvim.colorscheme
-  -- pcall(vim.cmd.colorscheme, lvim.colorscheme)
-
-  if M._reload_triggered then
-    Log:debug "Reloaded configuration"
-    M._reload_triggered = nil
-  end
-end
+-- TODO: convert to lazy.nvim
```

Comment:
This might be the relevant event to use:
![image](https://user-images.githubusercontent.com/72631767/209899272-c4e2df8f-9ef0-499f-99db-775d5da25a88.png)

Reply:
the event would be a combination of LazyInstall and LazyUpdate, but these lines were commented out earlier and I'm not sure what they're supposed to do so I'll leave it for now